opam/opam

```diff
@@ -1,32 +1,36 @@
 opam-version: "1.2"
+maintainer: "Yoichi Hirai <i@yoichihirai.com>"
+authors: "Yoichi Hirai <i@yoichihirai.com>"
 name: "bamboo"
 version: "0.0.03"
-maintainer: "Yoichi Hirai <i@yoichihirai.com>"
-authors: [ "Yoichi Hirai <i@yoichihirai.com>" ]
-license: "Apache-2.0"
 homepage: "https://github.com/pirapira/bamboo"
-dev-repo: "https://github.com/pirapira/bamboo.git"
 bug-reports: "https://github.com/pirapira/bamboo/issues"
+license: "Apache-2.0"
+dev-repo: "https://github.com/pirapira/bamboo.git"
 build: [
-  ["rm" "-f" "src/parse/parser.ml" "src/parse/parser.mli" "src/parse/lexer.ml"]
+  [
+    "rm"
+    "-f"
+    "src/parse/parser.ml"
+    "src/parse/parser.mli"
+    "src/parse/lexer.ml"
+  ]
   ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
   ["ocaml" "setup.ml" "-build"]
 ]
 install: ["ocaml" "setup.ml" "-install"]
-remove: [
-  ["ocamlfind" "remove" "bamboo"]
-]
 build-test: [
   ["ocaml" "setup.ml" "-configure" "--enable-tests"]
   ["ocaml" "setup.ml" "-build"]
   ["ocaml" "setup.ml" "-test"]
 ]
+remove: ["ocamlfind" "remove" "bamboo"]
 depends: [
   "batteries" {build}
   "cryptokit" {build & >= "1.12"}
-  ("hex" {build & = "1.0.0"} | "hex" {build & = "0.2.0"} | "hex" {build & = "0.1.0"})
-  ("menhir" {build & = "20151005"} | "menhir" {build & = "20150921"} | "menhir" {build & = "20150914"} | "menhir" {build & = "20141215"} | "menhir" {build & = "20140422"} | "menhir" {build & = "20130911"} | "menhir" {build & = "20130116"} | "menhir" {build & = "20120123"})
-  "ocamlbuild" {build}
+  "hex" {build & >= "0.1.0" & <= "1.0.0"}
+  "menhir" {build & >= "20120123" & <= "20151005"}
+  "ocamlbuild" {build & (>= "0.9.3" | = "0")}
```

Comment:
This was necessary to avoid a bug in OCaml 4.03.0 where `-o` option didn't work.